### ISSUE TITLE
unable to share files or folders

### ISSUE THREAD

**Issue by L0ric0** [nextcloud/server#19908]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.

### Steps to reproduce
1. click on shareicon for a file or folder
2. a empty tab in the details sidepannel opens without the share tab (but the space for it is there)
3. same if you just open the details pannel

### Expected behaviour
that the share tab is there

### Actual behaviour
no share tab, unable to share files

### Server configuration

**Operating system:**
5.5.7-arch1-1 #1 SMP PREEMPT Sat, 29 Feb 2020 19:06:02 +0000 x86_64 GNU/Linux

**Web server:**
nginx version: nginx/1.16.1

**Database:**
psql (PostgreSQL) 12.2

**PHP version:**
PHP 7.4.3 (cli) (built: Feb 18 2020 15:35:13) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.3, Copyright (c), by Zend Technologies

**Nextcloud version:** (see Nextcloud admin page)
18.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:**
updated from 18.0.1 

**Where did you install Nextcloud from:**
archlinux package repository

**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>
Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - admin_audit: 1.8.0
  - announcementcenter: 3.7.0
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.2
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.2.0
  - cookbook: 0.5.7
  - dav: 1.14.0
  - event_update_notification: 1.0.1
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_antivirus: 2.2.1
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - flowupload: 0.1.8
  - gpxpod: 4.1.1
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - nextcloud_announcements: 1.7.0
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - password_policy: 1.8.0
  - photos: 1.0.0
  - previewgenerator: 2.2.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - recommendations: 0.6.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - socialsharing_email: 2.0.0
  - support: 1.1.0
  - survey_client: 1.6.0
  - suspicious_login: 3.1.0
  - systemtags: 1.8.0
  - text: 2.0.0
  - theming: 1.9.0
  - timetracker: 0.0.42
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - viewer: 1.2.0
  - workflowengine: 2.0.0
Disabled:
  - encryption
  - files_external
  - files_external_onedrive
  - user_ldap
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "uraziel.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "18.0.2.2",
        "overwrite.cli.url": "https:\/\/uraziel.de\/nextcloud",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "maintenance": false,
        "theme": "",
        "loglevel": 0,
        "log_type": "file",
        "data-fingerprint": "de75db130d06c80711abb2451c1c7753",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "app_install_overwrite": [
            "files_external_onedrive",
            "timetracker",
            "contacts"
        ],
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "ssl",
        "mail_sendmailmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465"
    },
    "apps": {
        "accessibility": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.4.0"
        },
        "activity": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "2.11.0"
        },
        "admin_audit": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "announcementcenter": {
            "types": "logging",
            "installed_version": "3.7.0",
            "enabled": "yes"
        },
        "backgroundjob": {
            "lastjob": "360"
        },
        "bruteforcesettings": {
            "types": "",
            "installed_version": "1.5.0",
            "enabled": "yes"
        },
        "calendar": {
            "types": "",
            "enabled": "yes",
            "installed_version": "2.0.2"
        },
        "cloud_federation_api": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.1.0"
        },
        "comments": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "contacts": {
            "types": "",
            "installed_version": "3.2.0",
            "enabled": "yes"
        },
        "cookbook": {
            "types": "",
            "enabled": "yes",
            "installed_version": "0.5.7"
        },
        "core": {
            "vendor": "nextcloud",
            "public_files": "files_sharing\/public.php",
            "public_webdav": "dav\/appinfo\/v1\/publicwebdav.php",
            "installedat": "1559660644.1448",
            "installed.bundles": "[\"CoreBundle\"]",
            "scss.variables": "08343dd3f787291558383b46e1b489fb",
            "theming.variables": "68361ecf33afeefbdb43dff3bf5c2f40",
            "updater.secret.created": "1569514034",
            "lastupdateResult": "[]",
            "backgroundjobs_mode": "cron",
            "lastupdatedat": "1584009636",
            "lastcron": "1584017470",
            "enterpriseLogoChecked": "yes",
            "oc.integritycheck.checker": "[]"
        },
        "dav": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.14.0",
            "regeneratedBirthdayCalendarsForYearFix": "yes",
            "buildCalendarSearchIndex": "yes",
            "chunks_migrated": "1",
            "buildCalendarReminderIndex": "yes"
        },
        "event_update_notification": {
            "types": "logging,dav",
            "enabled": "yes",
            "installed_version": "1.0.1"
        },
        "federatedfilesharing": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "federation": {
            "types": "authentication",
            "enabled": "yes",
            "installed_version": "1.8.0",
            "autoAddServers": "1"
        },
        "files": {
            "types": "filesystem",
            "enabled": "yes",
            "cronjob_scan_files": "500",
            "installed_version": "1.13.1"
        },
        "files_antivirus": {
            "av_mode": "socket",
            "av_socket": "\/var\/run\/clamav\/clamd.ctl",
            "av_host": "",
            "av_port": "0",
            "av_cmd_options": "",
            "av_infected_action": "only_log",
            "av_stream_max_length": "26214400",
            "av_max_file_size": "524288000",
            "av_path": "\/usr\/bin\/clamscan",
            "installed_version": "2.2.1",
            "types": "filesystem,dav",
            "enabled": "yes"
        },
        "files_external": {
            "types": "filesystem",
            "user_mounting_backends": "ftp,dav,owncloud,sftp,amazons3,swift,smb,files_external_onedrive,\\OC\\Files\\Storage\\SFTP_Key,\\OC\\Files\\Storage\\SMB_OC",
            "enabled": "no",
            "installed_version": "1.9.0"
        },
        "files_external_onedrive": {
            "types": "filesystem",
            "enabled": "no",
            "installed_version": "1.0.7"
        },
        "files_pdfviewer": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.7.0"
        },
        "files_retention": {
            "types": "filesystem",
            "enabled": "no",
            "installed_version": "1.7.0"
        },
        "files_rightclick": {
            "enabled": "yes",
            "types": "",
            "installed_version": "0.15.2"
        },
        "files_sharing": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.10.1"
        },
        "files_texteditor": {
            "installed_version": "2.8.0",
            "types": "",
            "enabled": "no"
        },
        "files_trashbin": {
            "types": "filesystem,dav",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "files_versions": {
            "types": "filesystem,dav",
            "enabled": "yes",
            "installed_version": "1.11.0"
        },
        "files_videoplayer": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.7.0"
        },
        "firstrunwizard": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "2.7.0"
        },
        "flowupload": {
            "types": "",
            "installed_version": "0.1.8",
            "enabled": "yes"
        },
        "gallery": {
            "types": "",
            "installed_version": "18.4.0",
            "enabled": "no"
        },
        "gpxpod": {
            "reset304": "1",
            "resetPics404": "1",
            "types": "",
            "enabled": "yes",
            "installed_version": "4.1.1"
        },
        "impersonate": {
            "types": "",
            "enabled": "no",
            "installed_version": "1.5.0"
        },
        "logreader": {
            "types": "",
            "enabled": "yes",
            "installed_version": "2.3.0",
            "levels": "01111"
        },
        "lookup_server_connector": {
            "types": "authentication",
            "enabled": "yes",
            "installed_version": "1.6.0"
        },
        "metadata": {
            "installed_version": "0.9.0",
            "types": "",
            "enabled": "no"
        },
        "nextcloud_announcements": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.7.0",
            "pub_date": "Thu, 24 Oct 2019 00:00:00 +0200"
        },
        "notifications": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "2.6.0"
        },
        "oauth2": {
            "types": "authentication",
            "enabled": "yes",
            "installed_version": "1.6.0"
        },
        "password_policy": {
            "types": "",
            "enabled": "yes",
            "enforceHaveIBeenPwned": "1",
            "installed_version": "1.8.0"
        },
        "photos": {
            "installed_version": "1.0.0",
            "types": "",
            "enabled": "yes"
        },
        "previewgenerator": {
            "installed_version": "2.2.0",
            "types": "filesystem",
            "enabled": "yes"
        },
        "privacy": {
            "types": "",
            "enabled": "yes",
            "readableLocation": "de",
            "installed_version": "1.2.0"
        },
        "provisioning_api": {
            "types": "prevent_group_restriction",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "recommendations": {
            "types": "",
            "enabled": "yes",
            "installed_version": "0.6.0"
        },
        "serverinfo": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "settings": {
            "installed_version": "1.0.0",
            "types": "",
            "enabled": "yes"
        },
        "sharebymail": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "socialsharing_email": {
            "installed_version": "2.0.0",
            "types": "",
            "enabled": "yes"
        },
        "support": {
            "enabled": "yes",
            "installed_version": "1.1.0",
            "types": "session"
        },
        "survey_client": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.6.0",
            "stats": "no",
            "files_sharing": "no",
            "encryption": "no",
            "last_sent": "1582749403",
            "last_report": "{\"id\":\"oc2h2sysu4v6\",\"items\":[[\"server\",\"version\",\"18.0.1.3\"],[\"server\",\"code\",\"other\"],[\"server\",\"enable_avatars\",\"yes\"],[\"server\",\"enable_previews\",\"yes\"],[\"server\",\"memcache.local\",\"\\\\OC\\\\Memcache\\\\APCu\"],[\"server\",\"memcache.distributed\",\"none\"],[\"server\",\"asset-pipeline.enabled\",\"no\"],[\"server\",\"filelocking.enabled\",\"yes\"],[\"server\",\"memcache.locking\",\"none\"],[\"server\",\"debug\",\"no\"],[\"server\",\"cron\",\"cron\"],[\"php\",\"version\",\"7.4.3\"],[\"php\",\"memory_limit\",1073741824],[\"php\",\"max_execution_time\",0],[\"php\",\"upload_max_filesize\",2097152],[\"database\",\"type\",\"pgsql\"],[\"database\",\"version\",\"PostgreSQL 12.1 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 9.2.0, 64-bit\"],[\"database\",\"size\",191284079],[\"apps\",\"files_texteditor\",\"disabled\"],[\"apps\",\"uploaddetails\",\"disabled\"],[\"apps\",\"viewer\",\"1.2.0\"],[\"apps\",\"contacts\",\"3.1.9\"],[\"apps\",\"gallery\",\"disabled\"],[\"apps\",\"accessibility\",\"1.4.0\"],[\"apps\",\"federatedfilesharing\",\"1.8.0\"],[\"apps\",\"files_pdfviewer\",\"1.7.0\"],[\"apps\",\"logreader\",\"2.3.0\"],[\"apps\",\"password_policy\",\"1.8.0\"],[\"apps\",\"privacy\",\"1.2.0\"],[\"apps\",\"provisioning_api\",\"1.8.0\"],[\"apps\",\"recommendations\",\"0.6.0\"],[\"apps\",\"serverinfo\",\"1.8.0\"],[\"apps\",\"support\",\"1.1.0\"],[\"apps\",\"survey_client\",\"1.6.0\"],[\"apps\",\"twofactor_backupcodes\",\"1.7.0\"],[\"apps\",\"updatenotification\",\"1.8.0\"],[\"apps\",\"federation\",\"1.8.0\"],[\"apps\",\"lookup_server_connector\",\"1.6.0\"],[\"apps\",\"oauth2\",\"1.6.0\"],[\"apps\",\"files\",\"1.13.1\"],[\"apps\",\"activity\",\"2.11.0\"],[\"apps\",\"dav\",\"1.14.0\"],[\"apps\",\"files_sharing\",\"1.10.1\"],[\"apps\",\"files_trashbin\",\"1.8.0\"],[\"apps\",\"files_versions\",\"1.11.0\"],[\"apps\",\"sharebymail\",\"1.8.0\"],[\"apps\",\"workflowengine\",\"2.0.0\"],[\"apps\",\"comments\",\"1.8.0\"],[\"apps\",\"firstrunwizard\",\"2.7.0\"],[\"apps\",\"nextcloud_announcements\",\"1.7.0\"],[\"apps\",\"notifications\",\"2.6.0\"],[\"apps\",\"systemtags\",\"1.8.0\"],[\"apps\",\"theming\",\"1.9.0\"],[\"apps\",\"announcementcenter\",\"3.7.0\"],[\"apps\",\"bruteforcesettings\",\"1.5.0\"],[\"apps\",\"metadata\",\"disabled\"],[\"apps\",\"impersonate\",\"disabled\"],[\"apps\",\"suspicious_login\",\"3.1.0\"],[\"apps\",\"files_rightclick\",\"0.15.2\"],[\"apps\",\"files_videoplayer\",\"1.7.0\"],[\"apps\",\"text\",\"2.0.0\"],[\"apps\",\"gpxpod\",\"4.1.1\"],[\"apps\",\"timetracker\",\"0.0.41\"],[\"apps\",\"files_external_onedrive\",\"1.0.7\"],[\"apps\",\"cloud_federation_api\",\"1.1.0\"],[\"apps\",\"files_external\",\"1.9.0\"],[\"apps\",\"admin_audit\",\"1.8.0\"],[\"apps\",\"event_update_notification\",\"1.0.1\"],[\"apps\",\"files_retention\",\"disabled\"],[\"apps\",\"previewgenerator\",\"2.2.0\"],[\"apps\",\"flowupload\",\"0.1.8\"],[\"apps\",\"cookbook\",\"0.5.7\"],[\"apps\",\"calendar\",\"2.0.1\"],[\"apps\",\"files_antivirus\",\"2.2.1\"],[\"apps\",\"photos\",\"1.0.0\"],[\"apps\",\"settings\",\"1.0.0\"]]}"
        },
        "suspicious_login": {
            "types": "authentication",
            "installed_version": "3.1.0",
            "enabled": "yes"
        },
        "systemtags": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "text": {
            "enabled": "yes",
            "types": "dav",
            "installed_version": "2.0.0"
        },
        "theming": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.9.0",
            "slogan": "***REMOVED SENSITIVE VALUE***",
            "url": "***REMOVED SENSITIVE VALUE***",
            "name": "Uraziel",
            "cachebuster": "3"
        },
        "timetracker": {
            "types": "",
            "installed_version": "0.0.42",
            "enabled": "yes"
        },
        "twofactor_backupcodes": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.7.0"
        },
        "updatenotification": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.8.0",
            "update_check_errors": "0",
            "timetracker": "0.0.42",
            "gpxpod": "4.1.1",
            "flowupload": "0.1.5",
            "suspicious_login": "3.1.0",
            "core": "17.0.2.1",
            "cookbook": "0.5.7",
            "calendar": "2.0.1",
            "event_update_notification": "1.0.1"
        },
        "uploaddetails": {
            "installed_version": "0.1.3",
            "enabled": "no",
            "types": ""
        },
        "viewer": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.2.0"
        },
        "workflowengine": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "2.0.0"
        }
    }
}
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
local

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:** 
Mozilla Firefox 74.0

**Operating system:**
5.5.8-arch1-1
#### Web server error log
<details>
<summary>Web server error log</summary>
none that have to do with nextcloud while reproducing the bug
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
unable to find logs that have to do with this only logline that could be interesting is:
Trying to access array offset on value of type null at /usr/share/webapps/nextcloud/3rdparty/leafo/scssphp/src/Compiler.php#5230
</details>

#### Browser log
<details>
<summary>Browser log</summary>

Console log

Content Security Policy: The page’s settings blocked the loading of a resource at eval (“script-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). common.js:2:319
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. addStylesClient.js:220:17
Proxying an event bus of version 1.1.2 with 0.2.1 ProxyBus.js:65:14
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 5 globals.js:66:15
OCA.Viewer initialized Viewer.js:41:10
OCA.Files.Sidebar initialized Sidebar.js:37:10
Content Security Policy: The page’s settings blocked the loading of a resource at eval (“script-src”). getInferredName.js:6:19
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
Proxying an event bus of version 1.1.2 with 0.2.1 ProxyBus.js:65:14
Proxying an event bus of version 1.1.2 with 0.2.1 ProxyBus.js:65:14
OCA.Sharing.ShareSearch initialized ShareSearch.js:33:10
OCA.Sharing.ExternalLinkActions initialized ExternalLinkActions.js:33:10
New search handler registered search.js:67:12
Clipboard is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
registerDetailView is deprecated! It will be removed in nextcloud 20. merged-index.js:7503:12
registerTabView is deprecated! It will be removed in nextcloud 20. merged-index.js:7492:12
Legacy tabs are deprecated! They will be removed in nextcloud 20. Tab.js:49:11
registerTabView is deprecated! It will be removed in nextcloud 20. merged-index.js:7492:12
Legacy tabs are deprecated! They will be removed in nextcloud 20. Tab.js:49:11
registerTabView is deprecated! It will be removed in nextcloud 20. merged-index.js:7492:12
Legacy tabs are deprecated! They will be removed in nextcloud 20. Tab.js:49:11
The humanFileSize library is deprecated! It will be removed in nextcloud 19. 13 globals.js:66:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
moment is deprecated: please ship your own, this will be removed in Nextcloud 20 6 globals.js:66:15
showDetailsView is deprecated! Use OCA.Files.Sidebar.activeTab. It will be removed in nextcloud 20. merged-index.js:4452:12
moment is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:66:15
The select2 library is deprecated! It will be removed in nextcloud 19. 4 globals.js:66:15
recommendations 
Array(3) [ {…}, {…}, {…} ]
SharingInput.vue:292
moment is deprecated: please ship your own, this will be removed in Nextcloud 20 32 globals.js:66:15
showDetailsView is deprecated! Use OCA.Files.Sidebar.activeTab. It will be removed in nextcloud 20. merged-index.js:4452:12
The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:66:15
The select2 library is deprecated! It will be removed in nextcloud 19. 4 globals.js:66:15
recommendations 
Array(3) [ {…}, {…}, {…} ]
SharingInput.vue:292
moment is deprecated: please ship your own, this will be removed in Nextcloud 20 25 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 287ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 163ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
showDetailsView is deprecated! Use OCA.Files.Sidebar.activeTab. It will be removed in nextcloud 20. merged-index.js:4452:12
XHRPROPFINDhttps://uraziel.de/nextcloud/remote.php/dav/files/lsteinert/.Contacts-Backup
[HTTP/2 207 Multi-Status 211ms]

The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
XHRPROPFINDhttps://uraziel.de/nextcloud/remote.php/dav/systemtags-relations/files/725
[HTTP/2 207 Multi-Status 122ms]

The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:66:15
The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
XHRPROPFINDhttps://uraziel.de/nextcloud/remote.php/dav/systemtags-relations/files/725
[HTTP/2 207 Multi-Status 151ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/apps/files_sharing/api/v1/shares?format=json&path=%2F.Contacts-Backup&reshares=true
[HTTP/2 200 OK 257ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/apps/files_sharing/api/v1/shares?format=json&path=%2F.Contacts-Backup&shared_with_me=true
[HTTP/2 200 OK 315ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/collaboration/resources/file/725?format=json
[HTTP/2 200 OK 361ms]

The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
XHRGEThttps://uraziel.de/nextcloud/ocs/v1.php/apps/files_sharing/api/v1/sharees_recommended?format=json&itemType=dir
[HTTP/2 200 OK 159ms]

recommendations 
Array(3) [ {…}, {…}, {…} ]
SharingInput.vue:292
moment is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 176ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:48:9
Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:667:10
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:1044:9
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:1051:9
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:1055:9
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:1307:10
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:175:9
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:182:9
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:186:9
Expected ‘none’, URL, or filter function but found ‘Alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:438:10
Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:132:11
Unknown property ‘-moz-border-radius’.  Declaration dropped. files:136:22
Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:146:11
Unknown property ‘-moz-border-radius’.  Declaration dropped. files:150:22
Unknown property ‘zoom’.  Declaration dropped. files:9:10
Expected declaration but found ‘*’.  Skipped to next declaration. files:10:5
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. files:55:13
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. files:71:13
Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:137:13
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. files:204:13
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. files:309:13
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. files:321:13
Expected ‘none’, URL, or filter function but found ‘progid’.  Error in parsing value for ‘filter’.  Declaration dropped. files:585:13
Expected ‘none’, URL, or filter function but found ‘alpha(’.  Error in parsing value for ‘filter’.  Declaration dropped. files:15:10
GEThttps://uraziel.de/nextcloud/css/core/d6dc-81bb-server.css?v=68361ecf33afeefbdb43dff3bf5c2f40-91378c80-3
GEThttps://uraziel.de/nextcloud/css/core/d6dc-81bb-css-variables.css?v=68361ecf33afeefbdb43dff3bf5c2f40-91378c80-3
GEThttps://uraziel.de/nextcloud/apps/files_pdfviewer/css/style.css?v=6a154fe0-3
GEThttps://uraziel.de/nextcloud/css/notifications/783c-81bb-styles.css?v=68361ecf33afeefbdb43dff3bf5c2f40-783c3f18-3
GEThttps://uraziel.de/nextcloud/css/core/d6dc-81bb-results.css?v=68361ecf33afeefbdb43dff3bf5c2f40-91378c80-3
GEThttps://uraziel.de/nextcloud/css/files/9eeb-81bb-merged.css?v=68361ecf33afeefbdb43dff3bf5c2f40-9eeb65bb-3
GEThttps://uraziel.de/nextcloud/css/text/d233-81bb-icons.css?v=68361ecf33afeefbdb43dff3bf5c2f40-d233662f-3
GEThttps://uraziel.de/nextcloud/apps/files_rightclick/css/app.css?v=5e585c9e-3
GEThttps://uraziel.de/nextcloud/apps/gpxpod/css/style.css?v=737ea995-3
GEThttps://uraziel.de/nextcloud/css/files_sharing/c5f2-81bb-icons.css?v=68361ecf33afeefbdb43dff3bf5c2f40-c5f24915-3
GEThttps://uraziel.de/nextcloud/css/activity/6c1e-81bb-style.css?v=68361ecf33afeefbdb43dff3bf5c2f40-6c1e7ede-3
GEThttps://uraziel.de/nextcloud/css/icons/icons-vars.css?v=1584009557
GEThttps://uraziel.de/nextcloud/apps/accessibility/css/user-93f80ff81a5a8e1dbcfadd042433abd0
GEThttps://uraziel.de/nextcloud/apps/theming/styles?v=3
moment is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
Unrecognized at-rule or error parsing at-rule ‘@-ms-viewport’. d6dc-81bb-server.css:1:23482
Unknown pseudo-class or pseudo-element ‘-ms-input-placeholder’.  Ruleset ignored due to bad selector. d6dc-81bb-server.css:1:55033
Unrecognized at-rule or error parsing at-rule ‘@-o-keyframes’. 9eeb-81bb-merged.css:1:24725
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 1609ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/apps/notifications/api/v2/notifications
[HTTP/2 200 OK 529ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 3 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 229ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 3 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 193ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 194ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
showDetailsView is deprecated! Use OCA.Files.Sidebar.activeTab. It will be removed in nextcloud 20. merged-index.js:4452:12
XHRPROPFINDhttps://uraziel.de/nextcloud/remote.php/dav/files/lsteinert/.Contacts-Backup
[HTTP/2 207 Multi-Status 266ms]

The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
XHRPROPFINDhttps://uraziel.de/nextcloud/remote.php/dav/systemtags-relations/files/725
[HTTP/2 207 Multi-Status 145ms]

The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:66:15
The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
XHRPROPFINDhttps://uraziel.de/nextcloud/remote.php/dav/systemtags-relations/files/725
[HTTP/2 207 Multi-Status 246ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/apps/files_sharing/api/v1/shares?format=json&path=%2F.Contacts-Backup&reshares=true
[HTTP/2 200 OK 235ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/apps/files_sharing/api/v1/shares?format=json&path=%2F.Contacts-Backup&shared_with_me=true
[HTTP/2 200 OK 338ms]

XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/collaboration/resources/file/725?format=json
[HTTP/2 200 OK 386ms]

The select2 library is deprecated! It will be removed in nextcloud 19. 2 globals.js:66:15
XHRGEThttps://uraziel.de/nextcloud/ocs/v1.php/apps/files_sharing/api/v1/sharees_recommended?format=json&itemType=dir
[HTTP/2 200 OK 221ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
recommendations 
Array(3) [ {…}, {…}, {…} ]
SharingInput.vue:292
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 168ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 230ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
XHRGEThttps://uraziel.de/nextcloud/ocs/v2.php/apps/notifications/api/v2/notifications
[HTTP/2 200 OK 177ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 195ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 3 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 194ms]

moment is deprecated: please ship your own, this will be removed in Nextcloud 20 2 globals.js:66:15
XHRPOSThttps://uraziel.de/nextcloud/apps/text/session/sync
[HTTP/2 200 OK 202ms]

</details>

**Comment by martintilma** [comment]
Sharing by using the app on Android still works. Documents are then also visible in the web interface. So this looks like an ui bug

**Comment by Eorlariel** [comment]
It works properly with Google Chrome Browser.

**Comment by zingarese** [comment]
I am having this same issue. I started hosting Nextcloud (18.0.2) two days ago and I wasn't able to share anything.

**Comment by ler0x** [comment]
**TLDR: Try disabling uBlock origin. Helps for me :/**

**First of all:**
I'm having the same issue on two seperate servers. Both running Nextcloud 18.0.3. Both running Arch linux.

Number One is on arch (5.5.4-arch1-1 #1 SMP PREEMPT Sat, 15 Feb 2020 00:36:29 +0000 x86_64 GNU/Linux) with nginx-only (1.16.1-2), PHP (7.4.2-1) and MariaDB (10.4.12-1).

Number Two is on arch (5.4.20-1-lts #1 SMP Sat, 15 Feb 2020 00:19:19 +0000 x86_64 GNU/Linux) with apache-only (2.4.41-1), PHP (7.4.2-1) and MariaDB (10.4.12-1).

**Some research:**
Since @martintilma already descirbed the issue as UI-Issue and for @Eorlariel it is working on Chrome i did some research with different browsers.

This research was done on server number two (see top) and at first with Nextcloud 18.0.1, but after updating to 18.0.3 the problem persists. It seems that there is no relevant difference between this two versions regarding this issue.

_My system:_
Windows 10 64-bit Version 1909 (Build 18363.720)

_I used three browsers:_
- Opera (since i'm refusing to get chrome, but it's also based on Chromium) with version: 67.0.3575.97
- Firefox Developer Edition (on beta-channel Aurora) with version: 75.0b8
- Firefox Browser (on stable-channel) with version: 72.0.2 & after updating: 74.0

**Result:**
The share-tab is invisible in some browsers, but the requried space is there all the time (see Picture). I also observed, that sometimes the tab will be visible in Opera and Firefox Dev on first load of the site, but then get invisible a second later.
Therefore i looked up the HTML code with development tools: The HTML code seem to be the same in all browsers, all HTML-tags regarding the tab for sharing are there and a quick look at the CSS code revealed: there is a "display: none !important; set for id="sharing" and a whole bunch of others".

**Explanation / Solution:**
In Opera and Firefox Dev Edition i use the browser-plugin "uBlock origin" and it seems to block every element with the id "sharing". Disabling the plugin makes the tab visible again.

Hopefully this is also the solution for you others, so i didn't waste my time. Then it would be great to change the HTML-tag-ID from "sharing" to something else.

![Nextcloud share](https://user-images.githubusercontent.com/10778307/77647806-60870f00-6f67-11ea-85a7-41e2274c9f73.png)

**Comment by MartinTrifork** [comment]
@ler0x Thanks for the hint. U use Adblock plus. After adding my nextcloud url to the whitelist the sharing tab was visible again.

**Comment by kesselb** [comment]
cc @skjnldsv @ChristophWurst I'm still not a fan of using weird class names to solve that kind of issue but using the web today without those tools is not an option either.

**Comment by skjnldsv** [comment]
> @ler0x Thanks for the hint. U use Adblock plus. After adding my nextcloud url to the whitelist the sharing tab was visible again.

Stop using adblock plus is my recommendation, https://www.reddit.com/r/privacy/comments/bvj5nj/use_ublock_origin_not_adblock/

> **TLDR: Try disabling uBlock origin. Helps for me :/**

I don't have this issue with uBlock origin.
What is the filter name responsible for this again?

**Comment by MartinTrifork** [comment]
removed Adblock and installed uBlock Origin. Sharing button is still available.

**Comment by jatherrien** [comment]
I was experiencing this bug with uBlock Origin; looking at uBlock Origin's logs it was blocked by the filter lists "Fanboy's Social" and "Fanboy's Annoyance" (which admittedly I don't believe are default in uBlock, but are provided for someone to enable if they desire). I believe this also somewhat impacts Nextcloud 17; I was having difficulty creating a public link (clicking it did nothing), which I tried to solve by upgrading to Nextcloud 18, causing this bug.

**Comment by skjnldsv** [comment]
So, they are literally blocking:
All `.icon-share` classes and any `#sharing`...
This is most likely breaking things in a lot of different places and website.

Closing as wontfix

**Comment by kolAflash** [comment]
Did someone open issues in the bug trackers of "Adblock Plus" and "uBlock Origin", or send this information to the maintainer of that "Fanboy’s ..." filterlist?

I can reproduce with "uBlock Origin", if I manually enable "Fanboy’s Social​​​​​" or "Fanboy’s Annoyance​​​​​​​​​".

But I can't reproduce with "Adblock Plus". Do I need to manually set something in the "Adblock Plus" settings?

**Comment by skjnldsv** [comment]
@kolAflash as far as I know, this only happens with manual enabling of external rules that are not shipped by default for ABP or uBlock

**Comment by kolAflash** [comment]
Thanks!
I just found out:

Reproduce uBlock Origin:
Enable `Fanboy’s Annoyance​​​​​​​​​` or `Fanboy’s Social​​​​​` (both are preconfigured, but not activated).

Reproduce Adblock Plus:
Enable setting `Block additional tracking`.

**Comment by krystian3w** [comment]
https://github.com/AdguardTeam/AdguardFilters/issues/51643#issuecomment-599787060
